### ISSUE TITLE
bug in typo with function get Cid

### ISSUE THREAD

**Issue by snowwolf007cn** [jpush/jpush-api-php-client#75]
```
    public function getCid($count = 1, $type = 'push') {
        $url = $this->client->makeURL('push') . 'push/cid?count=' . $count . '&type =' . $type;
        return Http::get($this->client, $url);
    }
```

there shouldn't be  a whitespace in '&type =', must be '&type='

**Comment by yeahlife** [comment]
我说呢，今天刚好，遇到这个bug,官方也不着急改，每次升级jpush总是这样那样的问题

**Comment by helperhaps** [comment]
fixed in v3.5.37,please update
